v5/core/utils.go

```diff
@@ -182,6 +188,25 @@ func GetCurrentTime() int64 {
 	return time.Now().Unix()
 }
 
+// ParseDate parses the specified RFC3339 full-date string (YYYY-MM-DD) and returns a strfmt.Date instance.
+func ParseDate(dateString string) (fmtDate *strfmt.Date, err error) {
+	formattedTime, err := time.Parse(strfmt.RFC3339FullDate, dateString)
+	if err != nil {
+		return nil, err
+	}
+	formattedDate := strfmt.Date(formattedTime)
+	return &formattedDate, nil
+}
+
+// ParseDateTime parses the specified date-time string and returns a strfmt.DateTime instance.
+func ParseDateTime(dateString string) (fmtDTime *strfmt.DateTime, err error) {
```

Comment:
I think it would be better for this method to return a `strfmt.DateTime` rather than a pointer to `strfmt.DateTime`.  This is perhaps a question of style, and thus somewhat subjective, but in my defense I will note that `strfmt.ParseDateTime` returns a `strfmt.DateTime`.